ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -235,14 +228,15 @@ module.exports = Backbone.AssociatedModel.extend({
       this.set('locationType', 'utmUps')
     }
     this.drawing = false
+    this.set('drawing', false)
```

Comment:
Is it really necessary to have `this.drawing = false` _and_ `this.set('drawing', false)` here? If the goal of setting `this.drawing` was to affect the model, it obviously didn't achieve it since we also have to do `this.set`. I'm just not sure if that was the goal or if `this.drawing` serves another purpose

Reply:
I wasn't sure either, it seems that `this.drawing` is checked in some places in the file - but maybe those checks could just be replaced with `this.get('drawing')`. I tried this and it seems to work fine so I'll push it up